### PR TITLE
Added Eldritch_Edge and Blighting_Brand NMs and stuff 2.0

### PR DESCRIPTION
**Pull request by Fiocitrine**
Ok so I didn't push a bunch of stuff this time with my added stuff. Sorry for the annoyance.

### CODE REVIEW

**Comment by xdemolish** [comment]
amend > descriptive commit message > force push

drop a comment when done

File: scripts/zones/Rolanberry_Fields/mobs/Eldritch_Edge.lua [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    -- Set Eldritch Edge's Window Open Time
+    local wait = math.random(5400,7200);
+    	SetServerVariable("[POP]Eldritch_Edge", os.time(t) + wait); -- 90 to 120 minutes
+    	DeterMob(mob:getID(), true);
+
+    -- Set PH back to normal, then set to respawn spawn
+    local PH = GetServerVariable("[PH]Eldritch_Edge");
+        SetServerVariable("[PH]Eldritch_Edge", 0);
+        DeterMob(PH, false);
+   		GetMobByID(PH):setRespawnTime(GetMobRespawnTime(PH));
```
Comment (TeoTwawki) [inline scripts/zones/Rolanberry_Fields/mobs/Eldritch_Edge.lua]: Taaaaaaaaaaaaaaabs

File: scripts/zones/Sauromugue_Champaign/mobs/Blighting_Brand.lua [hunk 2]
```diff
@@ -11,4 +11,15 @@ require("scripts/globals/fieldsofvalor");
...
+     -- Set Eldritch Edge's Window Open Time
+    local wait = math.random(5400,7200);
+    	SetServerVariable("[POP]Blighting_Brand", os.time(t) + wait); -- 90 to 120 minutes
+    	DeterMob(mob:getID(), true);
+
+    -- Set PH back to normal, then set to respawn spawn
+    local PH = GetServerVariable("[PH]Blighting_Brand");
+        SetServerVariable("[PH]Blighting_Brand", 0);
+        DeterMob(PH, false);
+   		GetMobByID(PH):setRespawnTime(GetMobRespawnTime(PH));
```
Comment (TeoTwawki) [inline scripts/zones/Sauromugue_Champaign/mobs/Blighting_Brand.lua]: Tabs again

**Comment by Fiocitrine** [comment]
fixed dem tabs and alignment stuffs. also fixed Lamina and hyakinthos drop rates.

**Comment by Fiocitrine** [comment]
Fixed that indent, love.

**Comment by xdemolish** [comment]
see my previous comment

**Comment by Fiocitrine** [comment]
"More recent "master" branch build has been stated." What does this mean?

File: scripts/zones/Rolanberry_Fields/mobs/Evil_Weapon.lua [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+-----------------------------------
+-- onMobDeath
+-----------------------------------
+
+function onMobDeath(mob,killer,ally)
+
+    local mobID = mob:getID();
+    if (Eldritch_Edge_PH[mobID] ~= nil) then
+        local ToD = GetServerVariable("[POP]Eldritch_Edge");
+        if (ToD <= os.time(t) and GetMobAction(Eldritch_Edge) == 0) then
```
Comment (xdemolish) [inline scripts/zones/Rolanberry_Fields/mobs/Evil_Weapon.lua]: t is undefined

Comment (xdemolish) [inline scripts/zones/Rolanberry_Fields/mobs/Evil_Weapon.lua]: os.time() will do

File: scripts/zones/Sauromugue_Champaign/mobs/Evil_Weapon.lua [hunk 4]
```diff
@@ -10,5 +10,19 @@ require("scripts/globals/fieldsofvalor");
 -----------------------------------
 
 function onMobDeath(mob,killer,ally)
+
     checkRegime(ally,mob,100,2);
+
+    local mobID = mob:getID();
+    if (Blighting_Brand_PH[mobID] ~= nil) then
+        local ToD = GetServerVariable("[POP]Blighting_Brand");
+        if (ToD <= os.time(t) and GetMobAction(Blighting_Brand) == 0) then
```
Comment (xdemolish) [inline scripts/zones/Sauromugue_Champaign/mobs/Evil_Weapon.lua]: same here

**Comment by xdemolish** [comment]
still want a descriptive commit message, though

**Comment by Fiocitrine** [comment]
Ok, I can do that. But I'm still unsure what you mean by "descriptive."

**Comment by teschnei** [comment]
your commit message says "Gah noobs"

There is no way you think that is descriptive

**Comment by Fiocitrine** [comment]
Of course not. I wasn't sure for what you were looking, exactly. Considering you guys are particular about what goes into the database, that's all.

**Comment by Fiocitrine** [comment]
I added the NMs Eldritch Edge and Blighting Brand and a way to spawn them. Also gave a spell list for Blighting Brand. Fixed the Lamina and Hyakinthos droprate for their items to reflect retail.

**Comment by xdemolish** [comment]
rebase / squash > use that shiny new descriptive commit message > force push

**Comment by Fiocitrine** [comment]
I'll fix that.

**Comment by teschnei** [comment]
okay, I thought you were thinking the pull request name was the commit message :p

the pull request name is pretty good - just like "Added Eldritch Edge and Blighting Brand NM" is fine
The commit message should just say what was done - you can check the messages for any of mine to see examples

**Comment by Fiocitrine** [comment]
Ok there we go. I rebased and it renamed the commit? Things are alright, for now. 

I added the NMs Eldritch Edge and Blighting Brand and a way to spawn them. Also gave a spell list for Blighting Brand. Fixed the Lamina and Hyakinthos droprate for their items to reflect retail.

File: scripts/zones/Rolanberry_Fields/mobs/Eldritch_Edge.lua [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+-----------------------------------
+-- onMobDeath
+-----------------------------------
+
+function onMobDeath(mob,killer,ally)
+
+    -- Set Eldritch Edge's Window Open Time
+    local wait = math.random(5400,7200);
+    SetServerVariable("[POP]Eldritch_Edge", os.time(t) + wait); -- 90 to 120 minutes
```
Comment (xdemolish) [inline scripts/zones/Rolanberry_Fields/mobs/Eldritch_Edge.lua]: whoops missed this one

File: scripts/zones/Sauromugue_Champaign/mobs/Blighting_Brand.lua [hunk 6]
```diff
@@ -11,4 +11,15 @@ require("scripts/globals/fieldsofvalor");
 
 function onMobDeath(mob,killer,ally)
     checkRegime(ally,mob,100,2);
+
+     -- Set Eldritch Edge's Window Open Time
+    local wait = math.random(5400,7200);
+    SetServerVariable("[POP]Blighting_Brand", os.time(t) + wait); -- 90 to 120 minutes
```
Comment (xdemolish) [inline scripts/zones/Sauromugue_Champaign/mobs/Blighting_Brand.lua]: os.time() here too

File: scripts/zones/Rolanberry_Fields/mobs/Evil_Weapon.lua (hunk 3 above)
Comment (TeoTwawki) [inline scripts/zones/Rolanberry_Fields/mobs/Evil_Weapon.lua]: t is not undefined, its part of `os.time()`'s internal table:
http://www.lua.org/manual/5.1/manual.html#pdf-os.time
`t` tells it "give me the clock in seconds" 
https://github.com/DarkstarProject/darkstar/blob/0307e011f10bb108b6bbc6bf81731ea9f3e7608d/scripts/zones/Korroloka_Tunnel/mobs/Clipper.lua#L20
https://github.com/DarkstarProject/darkstar/blob/0307e011f10bb108b6bbc6bf81731ea9f3e7608d/scripts/zones/Korroloka_Tunnel/mobs/Cargo_Crab_Colin.lua#L14
And is correct here.

File: scripts/zones/Sauromugue_Champaign/mobs/Evil_Weapon.lua (hunk 4 above)
Comment (TeoTwawki) [inline scripts/zones/Sauromugue_Champaign/mobs/Evil_Weapon.lua]: still correct here

**Comment by TeoTwawki** [comment]
@xdemolish @fiocitrine `os.time(t)` was in fact correct. Its used this way all throughout existing scripts and is not undefined - its specifies which table os.time() is to return, in this case "clock in seconds".

**Comment by xdemolish** [comment]
wait shit i thought that was 't' not just t

**Comment by TeoTwawki** [comment]
Wtf github eating some of my comments

@xdemolish I had linked you the lua book and 2 instances of its use in DSP that just use t, github seems to have eaten that post. I dunno anything about 't' instead of t but if the DSP example I liked is wrong then most and DSP is wrong & lol at that.

**Comment by Fiocitrine** [comment]
Should I change it back? I'm confused...

**Comment by Fiocitrine** [comment]
Well I'm not getting this mob to spawn, and it worked with (t) so I'm just gonna set it back to that.

**Pull request merged**